ccc/elasticsearch.py

```diff
@@ -115,3 +117,30 @@ def store_document(
             *args,
             **kwargs,
         )
+
+    def store_bulk(
+        self,
+        body: str,
+        inject_metadata=True,
+        *args,
+        **kwargs,
+    ):
+        util.check_type(body, str)
+
+        if inject_metadata and _metadata_dict():
+            md = _metadata_dict()
+            file = body
```

Comment:
nit: I think it's a bit misleading to
a) call the copy 'file' (because it still is a string)
b) overwrite the argument passed-in

how about sth. like so:
```
def inject_meta(line):
  parsed = json.loads(line)
  if not 'index' in parsed:
    parsed['cc_meta'] = md
    return json.dumps(parsed)
  return line

patched_body = '\n'.join([inject_meta(line) for line in body.splitlines])
...
```